tests/acceptance/features/coreApiWebdavOperations/propfind.feature

```diff
@@ -20,19 +20,23 @@ Feature: PROPFIND
 
   Scenario Outline: send PROPFIND request to "/remote.php/dav/(files|spaces)" with depth header
     Given user "Alice" has been created with default attributes and without skeleton files
-    When user "Alice" requests "<dav_path>" with "PROPFIND" using basic auth and with headers
+    When user "Alice" requests "<dav-path>" with "PROPFIND" using basic auth and with headers
       | header | value   |
       | depth  | <depth> |
-    Then the HTTP status code should be "207"
+    Then the HTTP status code should be "<http-code>"
     Examples:
-      | dav_path                    | depth    |
-      | /remote.php/dav/files/alice | 0        |
-      | /remote.php/dav/files/alice | infinity |
+      | dav-path                    | depth    | http-code | 
+      | /remote.php/webdav          | 0        | 207       |
+      | /remote.php/webdav          | 1        | 207       |
+      | /remote.php/dav/files/alice | 0        | 207       |
+      | /remote.php/dav/files/alice | 1        | 207       |
 
     @skipOnRevaMaster
     Examples:
-      | dav_path                         | depth    |
-      | /remote.php/dav/spaces/%spaceid% | 0        |
+      | dav-path                         | depth    | http-code | 
+      | /remote.php/dav/spaces/%spaceid% | 0        | 207       |
+      | /remote.php/dav/spaces/%spaceid% | 1        | 207       |
+      | /remote.php/dav/spaces/%spaceid% | infinity | 400       |
```

Comment:
test cases for old/new dav paths will be added in this PR https://github.com/owncloud/ocis/pull/7397